### PR TITLE
Issue #27 Completing activation of the security TCK.

### PR DESCRIPTION
**Pull request by darranl**

### CODE REVIEW

**Comment by darranl** [comment]
FYI the failures are expected until we have a new release of WildFly.  

One failure still needs challenging to see if we can agree to ignore but we have discussed that the reason for the failure is not actually the purpose of that test.

File: security/run-tck.sh [hunk 1]
```diff
@@ -211,7 +212,7 @@ then
     echo "Configuring WildFly for the Old TCK"
     pushd $TS_HOME/bin
     $ANT_HOME/bin/ant config.vi
```
Comment (bstansberry) [inline security/run-tck.sh:214]: @darranl It seems the various uses of $ANT_HOME/bin is not necessary. If so we should clean that out having the mix of usage is confusing. That shouldn't block this though.

**Comment by jamezp** [comment]
@darranl Just an FYI. This currently runs tests against the latest successful build of the Nightly WildFly CI job. Should it be passing on last nights main branch?

**Comment by bstansberry** [comment]
@jamezp The nightly job last night had 1 failure and thus didn't publish the artifact. Last successful run was against https://github.com/wildfly/wildfly/commit/62f0e6762ca5a405a47b281693ab4add6e3aa00a which doesn't include https://github.com/wildfly/wildfly/pull/15968

**Pull request merged**